### PR TITLE
skaff: add ephemeral resources

### PR DESCRIPTION
**Pull request by johnsonaj**
### Output from Acceptance Testing

```console
% make testacc TESTS=TestAccXXX PKG=ec2

...
```

### CODE REVIEW

**Review by YakDriver — DISMISSED** [review]
LGTM 🎉 

```console
% cd skaff
% go install .
% cd ..
% cd internal/service/ecs
% skaff
Create scaffolding for the Terraform AWS Provider

Usage:
  skaff [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  datasource  Create scaffolding for a data source
  ephemeral   Create scaffolding for a ephemeral resource
  function    Create scaffolding for a function
  help        Help about any command
  resource    Create scaffolding for a resource

Flags:
  -h, --help   help for skaff

Use "skaff [command] --help" for more information about a command.
% skaff ephemeral
Error: error checking: no name given
Usage:
  skaff ephemeral [flags]

Flags:
  -c, --clear-comments     do not include instructional comments in source
  -f, --force              force creation, overwriting existing files
  -h, --help               help for ephemeral
  -n, --name string        name of the entity
  -s, --snakename string   if skaff doesn't get it right, explicitly give name in snake case (e.g., db_vpc_instance)
% skaff ephemeral -n Frank -f
%
```

**Review by YakDriver — APPROVED** [review]
Oh, neat! I can approve after changes.

File: skaff/ephemeral/ephemeraltest.gtpl [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+					statecheck.ExpectKnownValue(echoResourceName, dataPath.AtMapKey("secret_string"), knownvalue.StringExact(secretString)),
+				},
+			},
+		},
+	})
+}
+
+func testAcc{{ .EphemeralResource }}EphemeralResourceConfig_basic(rName, version string) string {
+	return acctest.ConfigCompose(
+	    acctest.ConfigWithEchoProvider("ephemeral.aws_{{ .ServicePackage }}_{{ .EphemeralResourceSnake }}.test"),
```
[the remark this answers is not in the document]
Comment (jar-b) [reply skaff/ephemeral/ephemeraltest.gtpl:201]: Verry coool that you've included everything to make working with echo easy!

> Verry coool that you've included everything to make working with echo easy!
>> Verry coool that you've included everything to make working with echo easy!

**Review by jar-b — DISMISSED** [review]
LGTM 🚀

**Pull request merged**